exercicios/para-casa/entregas/geisiane-de-oliveira/resolucao.js

```diff
@@ -0,0 +1,62 @@
+class Author{
+
+    #nameAuthor;
+    #email;
+    #birthdate;
+
+    constructor(nameAuthor, email, birthdate){
+        this.#nameAuthor = nameAuthor;
+        this.#email = email;
+        this.#birthdate = birthdate;
+    }
+
+}
+
```

Comment:
Oii Geisi, muito bom o seu código! Deu pra ver que você pegou a ideia do exercício bem;

Só senti falta dos métodos de acesso públicos (getters e setters) na classe Author que o exercício pedia, como:

get nameAuthor(){
    return this.#name;
}

Já que os atributos privados geralmente devem ser acessados pelos seus métodos.